src/__tests__/test_hello.js

```diff
@@ -1,14 +0,0 @@
-import expect from 'expect';
-
-describe('hello', () => {
```

Comment:
Good Job! This needed to go a while ago. 

Here's a list
 * You
 * Are
 * Awesome

How about a code block?

``` python
import this
```